src/coreclr/src/vm/excep.cpp

```diff
@@ -6703,6 +6707,9 @@ bool IsIPInMarkedJitHelper(UINT_PTR uControlPc)
     CHECK_RANGE(JIT_WriteBarrier)
     CHECK_RANGE(JIT_CheckedWriteBarrier)
     CHECK_RANGE(JIT_ByRefWriteBarrier)
+#if !defined(TARGET_ARM64)
```

Comment:
What's special about ARM64 that it is not in this list?

Reply:
ARM64 doesn't have the JIT_StackProbe (yet). See #13519.